package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "cognito-service-app",
   "description": "Typescript service for AWS Cognito with Angular 6",
-  "version": "1.2.4",
+  "version": "1.2.6",
```

Do not update the version for each commit but for each merge. So +1 is enough (1.2.5)
```suggestion
  "version": "1.2.5",
```

---

Done! 😄